app/javascript/components/upload/ExpandableFileForm.jsx

```diff
@@ -147,8 +153,8 @@ export function SaveDeleteButtons({
       onHide={() => setShowConfirmDeleteModal(false)}
       animation={false}>
       <Modal.Body className="">
-        Are you sure you want to delete {file.name}?<br />
-        <span>The file will be removed from the workspace and all corresponding database records deleted.</span>
+        Are you sure you want to delete {file.name}?<br /><br />
+        <span>All corresponding database records will be deleted. The file will <strong>{deleteText}</strong></span>
```

Comment:
Non-blocking, but the fact that we:

* needed to change this in 2 places (here and in `form-components.jsx`), and
* we observed drift in this PR

suggests that extracting this into one simple component would ensure efficient and consistent future changes.

Reply:
Agreed - I'm not sure why this is in two places but I didn't feel like dealing with it at the time.  Lemme see if I can refactor this without breaking anything.

Reply:
@eweitz [e5b0899](https://github.com/broadinstitute/single_cell_portal_core/pull/1963/commits/e5b0899373d71df80fce943c062355fcb3c7bc7e)